src/Algebra/Graph.hs

```diff
@@ -47,7 +47,10 @@ module Algebra.Graph (
     compose, box,
 
     -- * Context
-    Context (..), context
+    Context (..), context,
+
+    -- * Deforestation
+    buildg
```

Comment:
I suggest placing it into the same category as `foldg`: these two functions are related and it would be nice if they appear in the documentation together.